src/app/content/components/Toolbar/index.tsx

```diff
@@ -94,13 +100,19 @@ class Toolbar extends React.Component<Props, State> {
     const hideFromFocus = this.props.tocOpen === true
       || (this.props.tocOpen === null && !this.props.searchSidebarOpen);
 
+    const searchButtonColor = this.props.searchButtonStyle === 'grayButton' ? ('gray' as BookWithOSWebData['theme']) : (
+      this.props.searchButtonStyle === 'bannerColorButton' ? this.props.bookTheme : null
```

Comment:
i feel a little weird about allowing the third variant to fall through the null case, but its probably fine